osu.Game/Overlays/Toolbar/ToolbarButton.cs

```diff
@@ -157,25 +157,27 @@ protected ToolbarButton()
             };
         }
 
-        private readonly Cached tooltipKeyBinding = new Cached();
+        private RealmKeyBinding realmKeyBinding;
 
-        [BackgroundDependencyLoader]
-        private void load()
+        protected override void LoadComplete()
         {
-            keyBindings.KeyBindingChanged += () => tooltipKeyBinding.Invalidate();
-            updateKeyBindingTooltip();
-        }
+            base.LoadComplete();
 
-        private void updateKeyBindingTooltip()
-        {
-            if (tooltipKeyBinding.IsValid)
-                return;
+            if (Hotkey != null)
+            {
```

Comment:
would probably invert and early return in this method twice to de-nest it a little bit

Reply:
I guess. Doesn't really lend itself to running other stuff in this method in the future (which is why I avoid doing it in `LoadComplete` style methods).

Reply:
Alternative would be to split methods then maybe? Not sure it's a huge deal.